config/components/webhook/manifests.yaml

```diff
@@ -159,6 +159,7 @@ webhooks:
     apiVersions:
     - v1
     operations:
+    - CREATE
```

Comment:
I think we are also missing this for other integrations. Could you extend the fix, or follow up PRs? Also, for a follow up, it would be good to add an integration test for this to prevent regression.